### ISSUE TITLE
no sessions for oai-pmh requests

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/xcmetadataservicestoolkit#702]
JIRA issue created by: banderson
Originally opened: 2011-05-17 05:18 PM

Issue body:
 (nt)

**Comment by patrickzurek** [comment]
JIRA Coment by user: rcook
JIRA Timestamp: 2011-05-19 03:48 PM

Comment body:

Currently sessions are created auto by tomcat but we don't use them which is a system resource waste.  Consider changing this.

**Comment by patrickzurek** [comment]
JIRA Coment by user: banderson
JIRA Timestamp: 2011-06-09 07:04 PM

Comment body:

it turns out that sessions are not created for oai-pmh requests.  First I noticed that a cookie wasn't being sent to the client:

$ curl -s -vv 'http://localhost:8080/OAIToolkit/oai-request.do?verb=ListSets' > /dev/null
...
< HTTP/1.1 404 Not Found
< Server: Apache-Coyote/1.1
< Content-Type: text/html;charset=utf-8
< Content-Length: 1030
< Date: Thu, 09 Jun 2011 19:01:29 GMT

I was still curious if, perhaps, a session was being created for each request (even though the server sent back no cookie).  In order to check this, I added jmx to my tomcat:

-Dcom.sun.management.jmxremote
-Dcom.sun.management.jmxremote.port=9014
-Dcom.sun.management.jmxremote.ssl=false
-Dcom.sun.management.jmxremote.authenticate=false

and I added a new ant file to make a jmx request to say how many active sessions there are:

$ ant -file jmx-build.xml
...
     [echo]            sessions: 1

This process has verified that new sessions are not created for oai-pmh requests.
